File: Garden - Cookie Clicker/grid.py
class Grid:
    def __init__(self, x, y, elem=None):
        self.dimensions = (x,y)
        self.area = x*y

        self.grid = [[elem for x1 in range(x)] for y1 in range(y)]

    def set(self, x, y, elem):
        self.grid[y][x] = elem

    def get(self, x, y, elem):
        return self.grid[y][x]

    def __repr__(self):
        string = ""

        for horizarr in self.grid:
            string += "[" + " , ".join(horizarr) + "]\n"

        return string

File: Garden - Cookie Clicker/js/src/utils/grid.js
export class Grid {
    constructor(x, y, elem=null){
        this.dimensions = {x: x, y: y};
        this.area = x*y;
        this.grid = new Array(y);

        for(let i = 0; i < y; i++){
            this.grid[i] = new Array(x);
            this.grid[i].fill(elem);
        }
    }

    set(x, y, elem){
        this.grid[y][x] = elem;
    }

    get(x,y){
        return this.grid[y][x];
    }

    render(){
        // to be implemented
    }
}